engine/runtime-parser/src/test/java/org/enso/compiler/core/EnsoParserTest.java

```diff
@@ -447,7 +447,7 @@ public void testAssignments() throws Exception {
   }
 
   @Test
-  public void testNumberTimes() throws Exception {
+  public void testNumberTimes() {
```

Comment:
Can you avoid changing unrelated code?

Reply:
I follow [Boy Scout Rule](https://wiki.c2.com/?BoyScoutRule)